src/main/resources/db/migration/V1.0.2__init_enums.sql

```diff
@@ -0,0 +1,34 @@
+DROP TYPE IF EXISTS task_state_enum;
+CREATE TYPE task_state_enum as ENUM (
+    'UNCONFIGURED',
+    'PENDING_AUTO_ASSIGN',
+    'ASSIGNED',
+    'UNASSIGNED',
+    'COMPLETED',
+    'CANCELLED',
+    'TERMINATED',
+    'PENDING_RECOMMENDATION');
```

Comment:
should this be PENDING_RECONFIGURATION as per confluence documentation

Reply:
```suggestion
    'PENDING_RECONFIGURATION');
```